axon-server-connector/src/main/java/org/axonframework/axonserver/connector/ManagedChannelCustomizer.java

```diff
@@ -0,0 +1,21 @@
+package org.axonframework.axonserver.connector;
+
+import io.grpc.ManagedChannelBuilder;
+import java.util.function.UnaryOperator;
+
+/**
+ * Customizer to add more customizations to a managed channel to Axon Server.
+ *
+ * @author Marc Gathier
+ * @since 4.4.3
+ */
+public interface ManagedChannelCustomizer extends UnaryOperator<ManagedChannelBuilder<?>> {
```

Comment:
Curious: would you need to add `@FunctionalInterface` on this level too or is that taken over from the `UnaryOperator` automatically?

Reply:
Apparently not taken super interface, so added this.